src/main/java/graphql/execution/ValuesResolver.java

```diff
@@ -4,6 +4,7 @@
 import graphql.GraphQLContext;
 import graphql.Internal;
 import graphql.collect.ImmutableKit;
+import graphql.execution.values.InputInterceptor;
```

Comment:
There is a LOT of reformatting going on here.

This class AND `ValuesResolverCoercion` are a mess of inter call dependency

I found it really hard to navigate what method is being called and so on.

So I made the code multi line in parameter terms to help make it more readable

What you will see it `InputInterceptor` being passed around everywhere.  Some of the code (literal coercing say) don't need it BUT because they can in theory go into value coercion code, it needs to be put in place

Rather than refactor these two classes - I just passed it around everywhere so that it asked for once and then used.  For performance reasons I do NOT do a `graphqlContext.get()` at the inner method since I could do that per time its called.  Rather I asked for it once on the outer layers and then pass it everywhere.